plugins/copr.py

```diff
@@ -456,8 +462,9 @@ def _guess_chroot(self):
     def _download_repo(self, project_name, repo_filename, chroot=None):
         if chroot is None:
             chroot = self._guess_chroot()
-        short_chroot = '-'.join(chroot.split('-')[:2])
-        arch = chroot.split('-')[2]
+        index = 3 if len(self.chroot_parts) == 4 else 2
+        short_chroot = '-'.join(self.chroot_parts[:index])
+        arch = self.chroot_parts[index]
```

Comment:
I'd rather like to see something like:
```
# somewhat artificially we expect arch is on the last position
short_chroot = '-'.join(self.chroot_parts[:-1])
arch = self.chroot_parts[-1]
```

Reply:
But note that if you do just `dnf copr enable @copr/copr`, you get `self.opts.arg == ["@copr/copr"]` (len 1)
and the IndexError exception causes that `chroot_parts` is undefined here.

Reply:
Okay then. It can work with using `-1` as the index, the only question is what to do when len is 1.

Reply:
>  when len is 1

Not very nicely we return "Error: exactly two additional parameters to copr command are required" now :-(. We should at least say specify <project> [<chroot>] or something more friendly ... but that's yet another different issue.

Reply:
I made a change as suggested by praiskup. I took care of the situation when `dnf copr enable @copr/copr`